common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -87,6 +87,16 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+    val InizioSwitch = Switch(
+    Commercial,
+    "inizio",
+    "Include the Inizio script on page so that creatives can show a survey.",
+    owners = group(Commercial),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Should we have a review date, or is this a permanent feature?

Reply:
Good question! I assume this is a permanent feature, however that might change? Do we ever review commercial tags to see if it is still worth including/maintaining them?

@ScottPainterGNM also asking you while I still can 😉